_clubs/natural-areas-association-of-klahanie-naak.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Natural Areas Association of Klahanie (NAAK)
+date: 2019-07-01 07:14
+author: Klahanie
+comments: true
+categories: [clubs]
+link: true
+description: NAAK are careful stewards of Klahanie's natural areas
+thumbnail: /images/naak/Logo.jpg    
+banner: /images/naak/NAAK-cropped.jpg
+show-title: true
+---
+NAAK is a volunteer group of Klahanie residents committed to keeping Klahanie's natural areas healthy while providing a diverse habitat that protects and preserves wildlife.
+<img src="{{site.url}}images/naak/Logo.jpg" class="float-right">
+
+### What's happening now
+
+* The Native Plant Demonstration Garden (across from Klahanie Park) is undergoing rejuvenation. Plant labels have been added and selective pruning has provided improved definition between the plants.  Over 50 sword ferns were donated by the Sammamish Native Plant Stewards and planted for diversity.  Soon, additional mulch will be spread to retard the growth of weeds and grasses. Spring and summer will be an excellent time to visit the garden and enjoy the plant varieties and blooms.
+* Many projects will soon be underway to control noxious weeds in our natural areas. Noxious weeds can decrease the biodiversity of native ecosystems, reduce water quality and threaten wildlife habitat.  These plants tend to reproduce quickly, lack predators, and are difficult to control without a dedicated and ongoing commitment of stewardship.

```

Comment:
For consistency, both lists in this paragraph should use the Oxford comma - or not. "reduce water quality, and threaten wildlife"